### ISSUE TITLE
listen is MobilePay aplication installed;

### ISSUE THREAD

**Issue by Mactub07** [madsleejensen/react-native-mobilepay#3]
think of a situation : user doesn't have MobilePay app installed --> he click to button --> see custom alert with redirect to AppStore --> installed MobilePay --> then comeback to base application.
Problem : constant "MobilePay.isMobilePayInstalledFinland" (or for other countries) is not changes dinamycally. does exist way to listen installation of mobilePay app ( use not constant but callback)?

**Comment by madsleejensen** [comment]
yeah i thought about that, but decided to go with a simple "sync" up-front constants solution instead, looking back maybe this wasnt the best choice! Would be happy to accept a PR to provide async callback for checking if mobilepay is available.

**Comment by RasmusFredensborg** [comment]
I've made a pull request that adds support for this functionality: #5 :-)

**Comment by Vasant-Patel** [comment]
Another option would be to use
`const isInstalled = await Linking.canOpenURL('mobilepayfi://');`

**Comment by tknuts** [comment]
Hey Mads..
Will this React Native component run with Expo(expo.io)?
Regards, Thomas
